src/site/docs/synonyms/assets/es6-samples.xml

```diff
@@ -0,0 +1,1038 @@
+<?xml version="1.0"?>
+<synonyms>
+<theme id="theme-getting-started">
+  <title>Getting started</title>
+  <synonym id="syn-code-embedding">
+    <title>Code embedding</title>
+    <code language="javascript"><![CDATA[
+<script src='program.js'></script>
+]]></code>
+  </synonym>
+  <synonym id="syn-entry-point">
+    <title>Entry point</title>
+    <code language="javascript"><![CDATA[
+// Not required.
+function main() {
+  // To be used as the entry point, but it must be
+  // called manually.
+}
+
+main();
+
+// Sometimes the entry point is written as an
```

Comment:
note: this is typical done for scope protection (to prevent leaking into the global scope), but shouldn't be needed with ES6 modules. In ES6 you can just write top-level (module-level) code, and none of your local variables will be exported unless you say "export".

That said, modules aren't widely available yet. It's mainly the polyfills that implement them.